backend/tournaments/admin.py

```diff
@@ -3,34 +3,57 @@
 from .models import *
 
 class TournamentPlayerAdmin(admin.ModelAdmin):
-  list_display = ('tournament', 'member')
-
-
-class BucketPlayerAdmin(admin.ModelAdmin):
-  list_display = ('tournament_name', 'bucket_name', 'member')
+  list_display = ('member', 'tournament_name', 'bucket_name')
+  list_filter = ('bucket', 'member')
   def tournament_name(self, obj):
     return str(obj.bucket.tournament)
   def bucket_name(self, obj):
     return str(obj.bucket.name)
 
+class RoundAdmin(admin.ModelAdmin):
+  list_display = ('tournament', 'start')
+  list_filter = ('tournament__name', 'start')
```

Comment:
Yes. There it's a function name because foreign key "traversals" are not permitted in list_display (or I didn't find how to do that)
And for list_filter   tournament__name means  "tournament -> name" model traversal using foreign key (and functions are not permitted).